src/components/molecules/Button.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { TouchableHighlight, StyleSheet, Platform } from 'react-native'
+import Label from '../atoms/labels/Label';
+
+type Props = {
+  name: string,
+  result: string,
+  text: string,
+  width: number,
+  height: number,
+  fontSize: number,
+  color: string,
+  //onPress: () => {}
+}
+
+export const Button = (props: Props) => {
+
+  const { name, result, text, width, height, fontSize, color } = props
+
+  function onButtonPress() {
```

Comment:
This function should come as a callback property

Reply:
Oliver the onPress property is added to Button and use inside with the TouchableHighLight